src/envo/envo-edit.owl

```diff
@@ -3637,6 +3688,7 @@ Declaration(Class(<http://purl.obolibrary.org/obo/PO_0000003>))
 Declaration(Class(<http://purl.obolibrary.org/obo/PO_0009005>))
 Declaration(Class(<http://purl.obolibrary.org/obo/PO_0025025>))
 Declaration(Class(<http://purl.obolibrary.org/obo/PO_0025131>))
+Declaration(ObjectProperty(<http://purl.obolibrary.org/obo/BFO_0000051>))
```

Comment:
Not sure why these would be added, they already exist. 

There are a number of insert/deletions that seem superfluous - likely harmless, but please confirm you're using the latest version of Protege

Reply:
I am getting rid of this declaration now, but I notice that there are a large number of these in ENVO - unnecessary declarations. Perhaps good to clean up sep. from this PR.

Reply:
(@laurenechan is using the right Protege version - this is an artefact of the ROBOT template -> merge pipeline).

Reply:
> this is an artefact of the ROBOT template -> merge pipeline).
@matentzn good to know sorry if I inadvertently caused this issue with the pipeline. Is there anything you can suggest to help with this? or is it more like a merge conflict between this PR and recently merged the template/merge PR?

Reply:
Please dont cry in sadness when you see how I remove forgein entities from HPO: https://github.com/obophenotype/human-phenotype-ontology/blob/3cb4ef8d9da2287b567ceb4862c8476cec9083d1/src/ontology/hp.Makefile#L97 (you can tweak for ENVO). 
There are other ways, like doing robot remove --select ENVO: --axioms Declaration (I think something like this should work, not tested). Its nothing important really.. Dont worry too much.